schema/setup/control_50_servers_cache.cql

```diff
@@ -0,0 +1,14 @@
+USE @@KEYSPACE@@;
+
+CREATE TABLE servers_cache (
+    tenant_id ascii,
+    group_id ascii,
+    last_fetch timestamp,
+    server_id ascii,
+    server_blob ascii,
+    PRIMARY KEY((tenant_id, group_id), last_fetch, server_id)
+) WITH CLUSTERING ORDER BY (last_fetch DESC, server_id ASC) AND 
```

Comment:
Non-blocking - do we need `server_id` to be sorted?  (Also, does cass do sort it in ascending order?  I tried specifying `last_fetch` only in the clustering order - cass seems to return data with `server_id` sorted in ascending order, but I don't know if it's stored that way or if specifying the `server_id` does something special.  Sorry, I've forgotten since the last time we discussed sort order.  :(  ).